src/os/unix/ngx_getaffinity.c

```diff
@@ -0,0 +1,22 @@
+
+/*
+ * Copyright (C) Nginx, Inc.
```

Comment:
Is this feature originally from nginx? If not, we need to change this copyright to the right one.